portfolio/src/main/webapp/academics.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+  <body style="background c">
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+
+   <ulsidebar>
```

Comment:
Nice! Interesting choice. Why did you decide to create a custom element here instead of using `<div class="ulsidebar">`? 

I can't find the google standard, but it seems like an external standard is to use a hyphen in the custom element naming. `<ul-sidebar>` https://stackoverflow.com/q/5682943

Reply:
Functionally they both seemed to do the same thing so it didn't even cross my mind to make the new div. But I'm going to do that to conform to the standards

Reply:
What you had was fine, just more for advanced uses.